src/VisualStudio/Core/Def/Implementation/AbstractEditorFactory.cs

```diff
@@ -102,7 +88,7 @@ int IVsEditorFactory.Close()
             // to properly handle multiple windows open for the same document.
             if (punkDocDataExisting != IntPtr.Zero)
             {
-                object docDataExisting = Marshal.GetObjectForIUnknown(punkDocDataExisting);
+                var docDataExisting = Marshal.GetObjectForIUnknown(punkDocDataExisting);
```

Comment:
:memo: Still seeing a bunch of unrelated formatting changes in PRs. Please submit them as a separate PR to unify the style before working, or disable the editor feature and omit them.

Reply:
I am dogfooding code cleanup :)
 will use different PR next time